echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/postprocessors/ExpectedArtifactExpressionEvaluationPostProcessor.java

```diff
@@ -26,11 +29,22 @@
 @Component
 public class ExpectedArtifactExpressionEvaluationPostProcessor implements PipelinePostProcessor {
   private final ObjectMapper mapper;
-  private final ExpressionParser parser = new SpelExpressionParser();
+  private final ExpressionParser parser;
   private final ParserContext parserContext = new TemplateParserContext("${", "}");
 
-  public ExpectedArtifactExpressionEvaluationPostProcessor(ObjectMapper mapper) {
+  public ExpectedArtifactExpressionEvaluationPostProcessor(
+      ObjectMapper mapper, @Autowired ExpressionProperties expressionProperties) {
     this.mapper = mapper;
+    if (expressionProperties != null) {
```

Comment:
Without specifying required = false, I don't see how expressionProperies can be null.

Reply:
Updated the code.